lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -245,6 +245,16 @@ def process_assertions(message)
       process_filtered_fields(['assertion'], message)
     end
 
+    def user_info_attributes(user)
+      objects_hash = {'user'      => user,
+                      'tenant'    => user.current_tenant,
+                      'miq_group' => user.current_group}
+      objects_hash.each do |k, v|
+        value = MiqAeObject.convert_value_based_on_datatype(v.id, "#{v.class}")
+        @attributes[k] = value unless value.nil?
```

Comment:
1. Instead of `value.nil?`, should this be `v.nil?`, and not even assign `value` instead?
2. Would it make sense to shorten `MiqAeObject.convert_value_based_upon_datatype` to `MiqAeObject.convert`?
3. Does this need to be a loop?
4. `"#{v.class}"`  => `v.class.name`